FirebasePhoneAuthUI/CountryCode/FUICountryCodes.m

```diff
@@ -163,19 +154,58 @@ - (FUICountryCodeInfo *)countryCodeInfoForPhoneNumber:(NSString *)phoneNumber {
   return nil;
 }
 
-- (FUICountryCodes *)searchCountriesByName:(NSString *)countryName {
-  if ([countryName length] == 0) {
-    return nil;
-  } else {
-    NSArray *results =
-        [self.countryCodesArray filteredArrayUsingPredicate:
-            [NSPredicate predicateWithFormat:FUI_JSON_COUNTRY_NAME_PREDICATE, countryName]];
-    return [[FUICountryCodes alloc] initWithCountriesArray:results];
+- (void)blackListCountries:(NSSet<NSString *> *)countries {
+  NSMutableArray<NSDictionary *> *array =
+      [[NSMutableArray alloc] initWithCapacity:_countryCodesArray.count];
+  for (NSDictionary *dict in self.countryCodesArray) {
+    NSString *countryCode = dict[FUI_JSON_COUNTRY_CODE_KEY];
```

Comment:
nit: These constants aren't defined in this file and may not have been named in this CL, but they should be upper camel case with a preceding `k`. This change is optional, though.

`kFUIJSONCountryCodeKey`, `kFUIJSONDialCodeKey`

Reply:
Done